test/fixtures/flow/anonymous-function-no-parens-types/bad_01/actual.js

```diff
@@ -0,0 +1 @@
+var f = (x): number => 123 => 123;
```

Comment:
It would be nice to have a better error message that says something like `return type annotation needs to be wrapped in parens or have named parameters`. But not sure if this is easy to accomplish.
This should not be a blocker, just if you have an idea on how this could be done easy, otherwise we can come back to that in a separate story.